### DISCUSSION TITLE
Logos not rendering

### DISCUSSION THREAD

**Discussion by Chxresubles** [pradyunsg/furo#304] [Potential Issue] [answered]
In the latest version of furo, logos are not rendering correctly. The magnifying glass next to the search bar as well as icons next to infoboxes.
See attachements
![image](https://user-images.githubusercontent.com/54265171/141746791-1d894622-1ba1-4326-949a-22d9e7a3b518.png)
![image](https://user-images.githubusercontent.com/54265171/141746840-382b0db9-0ece-454b-a928-69e1bbbe0f48.png)

**Comment by pradyunsg** [comment]
What browser is this?

**Comment by Chxresubles** [comment]
I'm using Google Chrome Version 95.0.4638.69 (Official Build) (64-bit)

**Reply by Chxresubles** [reply]
Same thing happens on Microsoft Edge Version 95.0.1020.53 (Official build) (64-bit)

**Reply by pradyunsg** [reply]
Ah, right. Webkit has a custom browser prefix.

**Comment by pradyunsg** [comment, marked as answer]
Fixed in a5968632006d99ee4c3f699cc8950807c4d14855 and released as part of "Fearless Fawn". :)
